Lib/test/test_inspect.py

```diff
@@ -4235,10 +4235,30 @@ def func(*args, **kwargs):
         sig = inspect.signature(func)
         self.assertIsNotNone(sig)
         self.assertEqual(str(sig), '(self, /, a, b=1, *args, c, d=2, **kwargs)')
+
         func.__text_signature__ = '($self, a, b=1, /, *args, c, d=2, **kwargs)'
         sig = inspect.signature(func)
         self.assertEqual(str(sig), '(self, a, b=1, /, *args, c, d=2, **kwargs)')
 
+        func.__text_signature__ = '(self, a=1+2, b=4-3, c=1 | 4 | 16)'
+        sig = inspect.signature(func)
+        self.assertEqual(str(sig), '(self, a=3, b=1, c=21)')
```

Comment:
```suggestion
        func.__text_signature__ = '(self, a=1+2, b=4-3, c=1 | 3 | 16)'
        sig = inspect.signature(func)
        self.assertEqual(str(sig), '(self, a=3, b=1, c=19)')
```
Just because otherwise the test would pass if you had accidentally implemented `|` as `+`.